keyboards/xd60_rev3/xd60_rev3.c

```diff
@@ -0,0 +1,33 @@
+#include "xd60_rev3.h"
+
+
+extern inline void xd60_caps_led_on(void);
+extern inline void xd60_bl_led_on(void);
+
+extern inline void xd60_caps_led_off(void);
+extern inline void xd60_bl_led_off(void);
+
+
+void led_set_kb(uint8_t usb_led) {
+	// put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+
+  if (usb_led & (1<<USB_LED_CAPS_LOCK)) {
+		xd60_caps_led_on();
+	} else {
+		xd60_caps_led_off();
+  }
+
+  // if (usb_led & (1<<USB_LED_NUM_LOCK)) {
```

Comment:
You can remove this uncommented code if it's not being used.

Reply:
Of course!